ssl/record/ssl3_record.c

```diff
@@ -12,6 +12,7 @@
 #include <openssl/rand.h>
 #include "record_locl.h"
 #include "internal/cryptlib.h"
+#include <errno.h>
```

Comment:
ssl_locl.h is supposed to include errno.h, or am I missing something?

Reply:
Done